src/plus/github/github.ts

```diff
@@ -458,7 +458,7 @@ export class GitHubApi implements Disposable {
 
 			// If the pr is not from a fork, keep it e.g. show root pr's on forks, otherwise, ensure the repo owners match
 			const prs = rsp?.repository?.refs.nodes[0]?.associatedPullRequests?.nodes?.filter(
-				pr => !pr.repository.isFork || pr.repository.owner.login === owner,
+				pr => pr != null && (!pr.repository.isFork || pr.repository.owner.login === owner),
```

Comment:
Please undo this -- I'll check this in separately (as its unrelated to this changeset)

Reply:
Done. I face this exception quite frequently but this is handled already by try catch.